.github/workflows/rust.yml

```diff
@@ -51,11 +57,11 @@ jobs:
         with:
           name: build
       - name: Create app.zip
-        run: zip -rj app target/release/vln_node .codedeploy/*
+        run: zip -rj app target/release/vln_parachain genesis-state genesis-wasm .codedeploy/*
```

Comment:
Let's leave the `app.zip` as lean as possible with only what's needed for the instance deployment. This files will be pretty much useless inside a deployed instance. Instead upload the files separately. Reading the `aws s3 help` page I see they support uploading multiple files with `s3 mv . --recursive --exclude "*" --include "genesis-*" --include app.zip ...`. Or more accessible would be to upload as GH release, or .... do nothing! If we only need those files to manually download them to submit an extrinsic via polkadot.js/apps they should already be there in the `build` file from the previous step that we can access from the actions page.

Reply:
Oh I didn't know we could download the artefact from gh-actions, that's all we need 👍